Command/Tool/CalendarCommand.php

```diff
@@ -76,10 +76,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                         }
                         $time = sprintf('%02s:%02s', $parts[1], $parts[0]);
 
-                        $duplicates = array_filter($data, function ($row) use ($day, $time, $cron): bool {
-                            return $row[4]->getNextRunDate()->format('c') === $cron->getNextRunDate()->format('c');
-                            // return ($row[1] === $day && $row[3] === $time) || ($row[1] === '*' && $row[3] === $time) || ($day === '*' && $row[3] === $time);
-                        });
+                        $duplicates = array_filter(
+                            $data,
+                            fn (array $row): bool => $row[4]->getNextRunDate()->format('c') === $cron->getNextRunDate()->format('c')
+                            // fn (array $row): bool => ($row[1] === $day && $row[3] === $time) || ($row[1] === '*' && $row[3] === $time) || ($day === '*' && $row[3] === $time)
```

Comment:
Can be removed. Doesn't make sense to keep it if we remove `$time` and `$say` from the function signature. 👌